apps/frontend/src/pages/root/root.tsx

```diff
@@ -0,0 +1,7 @@
+import { AuthWrapper } from "~/libs/components/components.js";
+
+const Root: React.FC = () => {
```

Comment:
I don't like this solution. It means in future for any new screens we need to wrap in in AuthWrapper. We should keep AuthWrapper in ProtectedRoute and think of another way of dealing with the issue